internal/subshell/bash/bash.go

```diff
@@ -131,6 +140,20 @@ func (v *SubShell) SetupShellRcFile(targetDir string, env map[string]string, nam
 
 // SetEnv - see subshell.SetEnv
 func (v *SubShell) SetEnv(env map[string]string) {
+	for key, value := range env {
+		var bashified string
+		var err error
+		if key == "PATH" {
+			bashified, err = osutils.BashifyPathEnv(value)
+		} else {
+			bashified, err = osutils.BashifyPath(value)
+		}
+		if err != nil {
+			continue // ignore error because this probably isn't a path or path list to bashify
```

Comment:
This'll probably go away once line 149 is addressed but please actually assert this, don't rely on probability, or if you do; make sure it has a failsafe (ie. log it).

Reply:
Yes, this has gone away.